src/lib/sitemap-utils.ts

```
@@ -0,0 +1,71 @@
+        lastModified: lastModified || defaultDate,
+      }
+    })
+  )
+
+  return modificationTimes
+}
+
+/**
+ * Determine change frequency based on how recently the file was modified
+ */
+export function getChangeFrequency(lastModified: Date): 'always' | 'hourly' | 'daily' | 'weekly' | 'monthly' | 'yearly' | 'never' {
```

ESLint

[error] Replace `lastModified:·Date` with `⏎··lastModified:·Date⏎`

(prettier/prettier)